documentation/asciidoc/computers/camera_software.adoc

```diff
@@ -61,3 +61,7 @@ include::camera/v4l2.adoc[]
 include::camera/csi-2-usage.adoc[]
 
 include::camera/raspicam.adoc[]
+
+include::camera/synchronous_cameras.adoc[]
+
+include::camera/external_trigger.adoc[]
```

Comment:
I'm not sure about the ordering here.  Perhaps @aallan or @lurch might be able to advise?

Reply:
At the end seems reasonable at the moment. The legacy camera stack stuff is going away on the `bookworm` branch so as far as I remember the entire `raspicam.adoc` section disappears.